README.md

```diff
@@ -1,37 +1,66 @@
-Takeaway Challenge
-==================
+I'm very sorry to say that I tried to do my work this weekend on my laptop at home, and that I had serious problems with ruby version conflicts, stopping me from getting the twilio gem to work at all.
+
+Similarly, rspec wouldn't work, failing in a way that made no sense to me, so I tried to do what I could without it, and even that lead to problems that made me feel like I haven't learnt anything (eg I couldn't even initialize a class in irb).  All of this triggered a minor depressive episode and I decided to stop working before it got worse, try and get a good night's sleep and approach the next week's learning as positively as I can.  I'm very disappointed in myself and hope I can get past this, and apologise for my failure, and hope I've learnt at least not to bother trying to use my own laptop for work in future.
```

Comment:
Dude, don't sweat it. This shit is tough: knowing when to call it a day and take a health break is a skill in and of itself.